contracts/AirbroFactory.sol

```diff
@@ -116,10 +122,17 @@ contract AirbroFactory {
             tokenId,
             totalAirdropAmount,
             airdropDuration,
-            merkleRootHash
+            admin
         );
         airdrops.push(address(tokenDropContract));
         totalAirdropsCount++;
         emit NewAirdrop(rewardedNftCollection, address(tokenDropContract), msg.sender);
     }
+
+    /// @notice Updates the address of the admin variable
+    /// @param _newAdmin - New address for the admin of this contract, and the address for all newly created airdrop contracts
```

Comment:
Airdrop contracts are not using the admin address from the factory but you pass the same address from the factory in their constructor. That is a fine implementation and I think it is sufficient for this use case but also have in mind what you suggested about referencing from the airdrop contract and pulling the admin address from the factory so the Factory contract can in one transaction replace all admin addresses for all existing airdrop contracts.

Reply:
That's a great suggestion, definitively should be implemented